Dockerfile

```diff
@@ -12,6 +12,7 @@ WORKDIR /src
 ENV CASS_DRIVER_NO_EXTENSIONS theytaketoolongtobuild
 
 COPY requirements*.txt ./
+RUN apt-get -y update && apt-get -y install librdkafka-dev && rm -rf /var/lib/apt/lists/*
```

Comment:
I am not 100% sure anymore but I think it had smth to do with the apple silicon MBPs.

Reply:
Ah, interesting. I actually changed it to use x86_64 emulation in my PR (https://github.com/reddit/baseplate.py/pull/823/commits/97ff89d5bca69405e016d799c742fcb5ec697c6a) since there was no Thrift compiler image for ARM and it was failing on my MacBook trying to build a native arch version of the image. This is probably fine to keep as-is for now, though I'm surprised you didn't run into the same Thrift issue I did (maybe I'm doing something wrong?).

Reply:
```DOCKER_DEFAULT_PLATFORM=linux/amd64 docker-compose build```
worked locally for me

Reply:
so two different paths to achieve somewhat similar results :-D